### PR TITLE
chore: add an example showcasing OT vs different OTel setups

### PR DESCRIPTION
**Pull request by Rauno56**
# Todo

- [x] Add README

# Description

This adds an example instrumenting a toy express app and sending traces to Splunk APM without a collector in the middle.

- APMI-1923

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

- [x] Tested manually
- [ ] Added automated tests

# Checklist:

- [ ] Changelogs have been updated
- [ ] Unit tests have been added/updated
- [ ] Documentation has been updated

### CODE REVIEW

**Comment by Rauno56** [comment]
@theletterf, I just jotted down everything that the readme should, in my opinion, minimally include. I'm open to restructuring it or any other kind of feedback you might have.

**Review by theletterf — APPROVED** [review]
Let’s roll with it for now—we need a starting point.

File: examples/express/package.json [hunk 1]
```diff
@@ -0,0 +1,51 @@
+{
+  "name": "splunk-otel-example-express",
+  "private": true,
+  "version": "1.0.0",
+  "description": "Example of instrumenting a toy Express app in different ways",
+  "main": "server.js",
+  "scripts": {
+    "client": "env-cmd nodemon -r ./tracer.js client.js",
```
Comment (jtmalinowski) [inline examples/express/package.json:8]: So this `-r` is smart, but let's highlight it in the README. There's just one `server.js` file and it's not immediately obvious how it's instrumented.

File: examples/express/tracer.jaeger.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+const tracer = require('@splunk/otel').startTracing({
+	spanExporterFactory: (options) => {
+		const {
+			endpoint,
+			accessToken,
+		} = options;
+
+		return new JaegerExporter({
+	    endpoint: endpoint,
+		  username: 'auth',
```
Comment (jtmalinowski) [inline examples/express/tracer.jaeger.js:27]: is this because any username is required?

**Comment by jtmalinowski** [comment]
@Rauno56 So I'm wondering if we shouldn't have separate directories side-by-side for OpenTelemetry vs. OpenTracing. As a newcomer, would I be able to quickly understand which line of code belongs to which system?

File: examples/express/tracer.jaeger.js (hunk 2 above)
Comment (Rauno56) [reply examples/express/tracer.jaeger.js:27]: Not any. It literally has to be "auth".
https://github.com/signalfx/splunk-otel-js/blob/main/src/options.ts#L152-L153

**Comment by Rauno56** [comment]
> @Rauno56 So I'm wondering if we shouldn't have separate directories side-by-side for OpenTelemetry vs. OpenTracing. As a newcomer, would I be able to quickly understand which line of code belongs to which system?

Not sure what would we gain by separating them if they are actually exactly the same - only thing changes is the file we require before running our app.
I think is beneficial to see that in fact, the two solutions are effectively enabled the same way to be convincing in encouraging people to migrate.
+ it would be a burden to keep them in sync.

File: examples/express/package.json (hunk 1 above)
Comment (Rauno56) [reply examples/express/package.json:8]: Rewrote the overview section.

File: examples/express/SETUP.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
+
+The ingest setup is for OpenTracing only and doesn't mention access tokens
+
```
Comment (jtmalinowski) [inline examples/express/SETUP.md:3]: Is there a reason for this many empty lines?

**Review by jtmal-signalfx — APPROVED** [review]
Yeah so let's merge it, and keep improving based on feedback we'll get.

**Pull request merged**